bin/release_describer

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env ruby
+# Usage:
+# The first argument is the path to the directory containing the repos
+# The second argument is the previous release tag
+# bin/release_describer ~/Workspace v2023-11-08
+
+require "set"
+
+repos = ["nucore-nu", "nucore-osu", "nucore-umass", "nucore-dartmouth"]
+repo_dir = ARGV[0]
```

Comment:
What do you think about using current dir instead of getting this through an argument?

Reply:
good call!